### PR TITLE
Modify ui

### PR DESCRIPTION
**Pull request by turtle96**
Overall Changes
- Change window theme colours
- adjust font sizes to be bigger, more readable

### CODE REVIEW

**Comment by turtle96** [comment]
Please review
@CS2103AUG2016-W15-C3/testing 
@CS2103AUG2016-W15-C3/code-quality 
@CS2103AUG2016-W15-C3/documentation 

Think documentation might not need to check, since is mostly UI adjustments

File: src/main/java/seedu/taskell/ui/UiManager.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ void showAlertDialogAndWait(Alert.AlertType type, String title, String headerTex
     private static void showAlertDialogAndWait(Stage owner, AlertType type, String title, String headerText,
                                                String contentText) {
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/ui/UiManager.java:82]: Too deep indentation
image

**Comment by marcusngwenjian** [comment]
Code quality reviewed

> I like the colour

**Comment by marcusngwenjian** [comment]
Code quality reviewed

File: src/main/java/seedu/taskell/ui/UiManager.java (hunk 1 above)
Comment (turtle96) [reply src/main/java/seedu/taskell/ui/UiManager.java:82]: That isn't done by me I think ^^"'

**Pull request merged**